tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -60,14 +72,27 @@ static int s2n_setup_test_keys(struct s2n_connection *conn, struct s2n_config *c
     EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(config, ticket_key_name, strlen((char *)ticket_key_name),
                     ticket_key.data, ticket_key.size, current_time/ONE_SEC_IN_NANOS));
 
-    EXPECT_SUCCESS(s2n_connection_set_config(conn, config));
+    return S2N_SUCCESS;
+}
+
+static int s2n_setup_test_resumption_secret(struct s2n_connection *conn)
+{
+    POSIX_ENSURE_REF(conn);
+    /**
+     *= https://tools.ietf.org/rfc/rfc8448#section-3
+     *# PRK (32 octets):  7d f2 35 f2 03 1d 2a 05 12 87 d0 2b 02 41 b0 bf
+     *# da f8 6c c8 56 23 1f 2d 5a ba 46 c4 34 ec 19 6c
+     **/
+    S2N_BLOB_FROM_HEX(test_resumption_secret,
+    "7d f2 35 f2 03 1d 2a 05 12 87 d0 2b 02 41 b0 bf \
+         da f8 6c c8 56 23 1f 2d 5a ba 46 c4 34 ec 19 6c");
 
-    /* Set up master secret */
+    /* Set up resumption secret */
     struct s2n_blob secret = { 0 };
     struct s2n_stuffer secret_stuffer = { 0 };
-    EXPECT_SUCCESS(s2n_blob_init(&secret, conn->secure.master_secret, S2N_TLS_SECRET_LEN));
+    EXPECT_SUCCESS(s2n_blob_init(&secret, conn->resumption_master_secret, S2N_TLS_SECRET_LEN));
     EXPECT_SUCCESS(s2n_stuffer_init(&secret_stuffer, &secret));
-    EXPECT_SUCCESS(s2n_stuffer_write_bytes(&secret_stuffer, test_master_secret.data, test_master_secret.size));
+    EXPECT_SUCCESS(s2n_stuffer_write_bytes(&secret_stuffer, test_resumption_secret.data, test_resumption_secret.size));
```

Comment:
This all works, but couldn't you just do:
```
memcpy(conn->resumption_master_secret, test_resumption_secret.data, test_resumption_secret.size)
``

Reply:
That would be too easy Lindsay

Reply:
```POSIX_CHECKED_MEMCPY((conn->resumption_master_secret, test_resumption_secret.data, test_resumption_secret.size);``` :)